public/index.html

```diff
@@ -15,8 +15,20 @@
 
         <object id="floor1" type="image/svg+xml" data="assets/maps/ICF1Edit.svg"></object>
         <object id="floor2" type="image/svg+xml" data="assets/maps/ICF2Edit.svg"></object>
-
+        <div id="pins">
```

Comment:
If we have this in its own div, when we scale the webpage, the divs won't scale with the map graphic

A better alternative is to add the markers programmatically *inside* of the map SVG

Another thing I noticed is that when the map is switched from 1st to 2nd floor, the same markers stay in the same place, which shouldn't happen. If we programmatically add pins to the respective SVG, we'll avoid this issue